integration/spring-boot/starter-jaxws/pom.xml

```diff
@@ -69,6 +74,11 @@
             <artifactId>cxf-rt-frontend-jaxws</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This should be optional dependency as well

Reply:
I've changed it to compile scope.